docs/Glossary/index.md

```diff
@@ -0,0 +1,132 @@
+
+### Quality Definitions
+
+Defines the Minimum, Preferred, and Maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.
+
+### Quality Profile
+
+A defined group of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.
+
+---
+
+## Torrents
+
+### Cross-Seed
+
+The act of [seeding](#seeder) the same torrent across multiple [trackers](#tracker). It also refers to a software application of the same name that automates this process.
+
+### Deluge
+
+A lightweight, free, cross-platform BitTorrent client.
+
+### qBit Manage
+
+A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes, such as time and tracker.
+
+### qBittorrent
+
+A free, open-source, cross-platform BitTorrent client.
+
+### Leecher
+
+A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe users who download torrents without sharing them afterward.
+
+### Peer
+
+A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).
+
+### Seeder
+
+BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.
+
+### Swarm
+
+The collective group of all BitTorrent users connected to a given torrent, sharing parts of the file. This group includes both [Seeders](#seeder) and [Leechers](#leecher).
+
+### Torrent File
+
+A file containing metadata about the files and folders to be distributed, along with a list of network locations of [trackers](#tracker). Torrent files typically bear the extension `.torrent`.
+
+### Tracker
+
+Servers that assist participants in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.
+
+### Perma-Seed
+
+Seed (some of) your torrents permanently. Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places whilst not taking up extra hard disk space.
+
+---
+
+## Usenet
+
+### Atomic Moves
+
+This is a real file move, carried out instantaneously, not the copying of a file from the download folder to the media folder and then deleting the file from the download folder.
+
+### Instant Moves
+
+Same as [Atomic Moves](#atomic-moves).
+
+### DMCA
+
+Also known as: "DMCA takedown". The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.
+When this happens, you often end up with missing/incomplete parts and can't complete the download or repair it with PAR2.
```

Comment:
what does this have to do with DMCA?

Reply:
I understand what is being said, but I'm not sure this is relevant to the anchor section as a whole, perhaps this would be better suited in a repair/PAR section to explain the common use-case and failures of these files instead?

Reply:
> I understand what is being said, but I'm not sure this is relevant to the anchor section as a whole, perhaps this would be better suited in a repair/PAR section to explain the common use-case and failures of these files instead?

When you get to your review, would be good to add 'PAR' as a definition, and perhaps we can expand on it in another section of the guide.

Reply:
Sure.